platforms/codesandbox/prisma/.env

```diff
@@ -0,0 +1,2 @@
+CI=1
```

Comment:
This somehow also does not work :/

Reply:
Same, the container did get the correct `.env` file https://codesandbox.io/s/j6txg?file=/prisma/.env

Reply:
No events with `CI=true` for `%codesandbox%`:
https://app.redash.io/prismaio/queries/469381?p_ci=%5B%22false%22%2C%22true%22%5D&p_information=codesandbox

Reply:
Added comment to indicate this does not work.

Reply:
https://github.com/prisma/e2e-tests/issues/827